<launch>
    <node pkg="hrl_rfh_fall_2011" name="face_adls_manager" type="face_adls_manager.py"
          output="screen" />
    <!--
    <include file="$(find hrl_rfh_fall_2011)/launch/head_registration.launch" />
    <include file="$(find kelsey_sandbox)/launch/pr2_ar_servo.launch" />
    <node pkg="hrl_rfh_fall_2011" name="setup_arms_shaving" type="setup_arms_shaving.py"
          output="screen" args="-s" />
    <node pkg="hrl_rfh_fall_2011" name="setup_arms_servoing" type="setup_arms_servoing.py"
          output="screen" args="-s" />
    <node pkg="hrl_rfh_fall_2011" name="setup_registration" type="setup_registration.py"
          output="screen" args="-s" />
    <node pkg="pr2_traj_playback" name="traj_playback_r" type="arm_pose_move_controller.py"
          output="screen" args="-v -r" />
    <node pkg="pr2_traj_playback" name="traj_playback_l" type="arm_pose_move_controller.py"
          output="screen" args="-v -l" />
    -->

</launch>
